Classes/Domain/Repository/Hubspot/AccountInformationRepository.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+
+namespace T3G\Hubspot\Domain\Repository\Hubspot;
+
+use T3G\Hubspot\Domain\Repository\Hubspot\AbstractHubspotRepository;
+use T3G\Hubspot\Hubspot\Factory;
+
+/**
+ * Repository for Hubspot custom object definitions.
+ */
+class AccountInformationRepository extends AbstractHubspotRepository
+{
+    /**
+     * @param Factory|null $factory
+     */
+    public function __construct(Factory $factory = null)
+    {
+        parent::__construct($factory);
+    }
+
+    /**
+     * Get account details for a HubSpot account.
+     *
+     * @return array
```

Comment:
We don't need `@return` annotations if the returned type is declared already (`: array`).